### ISSUE TITLE
Screen reader is not announcing any information about the Sorting as ascending or descending in Details List.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14089]
## Issue: ###

Screen reader is not announcing any information about the Sorting as ascending or descending in Details List. 

### Test Environment: ###

Test Environment:
OS Build: Version 21H2(22000.613)
Browser: Chrome Version 100.0.4896.127 (Official Build) (64-bit) 
Screen reader: Non-Visual Desktop Access (NVDA) (2021.1)

### Repro steps: ###

1. Open above URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/customitemcolumns in Edge Dev browser and enable NVDA. 
2. Navigate till 'Key' column header present under 'Rendering Custom item column with sorting' table and activate it. 
3. Observe the issue with screen reader. 

### Actual result: ###

Screen reader is not announcing any information about the Sorting as ascending or descending in Details List. Screen reader remains silent after activating Key column header. 

### Expected result: ###

Screen reader should announce the information about the Sorting as ascending/descending in Details List after activating 'Key' column header as, "Sorted Ascending/Descending".
 
### Video: ###

https://user-images.githubusercontent.com/93735775/187549099-12f95c71-2c0e-40d5-8d79-6ca8b0e5d473.mp4

**Comment by josephsl** [comment]
Hi, wait, this is with Chrome 100? Isn’t the latest version Chrome 104? Thanks.

**Comment by mazen428** [comment]
It does read as sorted ascending for me, I'm on edge 104 and NVDA 2022.2

**Comment by rperez030** [comment]
For me, it works if I move away from the column header and come back to it in either focus mode or browse mode, but if I just press Enter or the space bar there is no change in speech or Braille.
NVDA 2022.2 installed and Edge 104.0.1293.70

**Comment by RohitJadhav-TCS** [comment]
Please find Codepen: [Fabric Example Pen (codepen.io)](https://codepen.io/codepen-io-kkrish051/pen/BaPLpPP)

**Comment by seanbudd** [comment]
Hi @RohitJadhav-TCS, this sample is written in a library we are not familiar with. Can you create a minimal plain **HTML** sample?
